script/generate_sql

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+This script generates SQL files for each SQL file in `sql/` and 
+includes all the temporary UDF definitions used in the queries.
+"""
+
+from argparse import ArgumentParser
+import os, sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
This `sys.path` manipulation is probably fine for the context of this repo, but can you add a comment to specify why this needs to happen and perhaps link to the StackOverflow answer that gives more context on the problem?

Would this go away if we moved `tests` underneath `bigquery_etl`?

Reply:
The problem would persist event if it's moved under `tests`